lldb-eval/parser.cc

```diff
@@ -2593,12 +2599,54 @@ lldb::SBType Parser::PrepareBinaryRemainder(ExprResult& lhs, ExprResult& rhs,
 }
 
 lldb::SBType Parser::PrepareBinaryBitwise(ExprResult& lhs, ExprResult& rhs,
+                                          bool is_shift_op,
```

Comment:
Let's make separate functions `PrepareBinaryBitwise` and `PrepareBinaryShift`. Right now the whole function is a big `if` with two branches.